### ISSUE TITLE
TUI Crashes when scrolling over specific post

### ISSUE THREAD

**Issue by TomCasavant** [ihabunek/toot#487]
I *think* the post that's causing the error is this https://mstdn.social/@game_music/112637746838855915. Basically whenever it hits that post the whole UI crashes, whether you're scrolling or click it directly. 

``` 
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/tom/.local/lib/python3.10/site-packages/toot/__main__.py", line 3, in <module>
    cli()
  File "/home/tom/.local/lib/python3.10/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/home/tom/.local/lib/python3.10/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/home/tom/.local/lib/python3.10/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/tom/.local/lib/python3.10/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/tom/.local/lib/python3.10/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/home/tom/.local/lib/python3.10/site-packages/toot/cli/__init__.py", line 124, in wrapped
    return f(get_context(), *args, **kwargs)
  File "/home/tom/.local/lib/python3.10/site-packages/toot/cli/tui.py", line 73, in tui
    tui.run()
  File "/home/tom/.local/lib/python3.10/site-packages/toot/tui/app.py", line 164, in run
    self.loop.run()
  File "/home/tom/.local/lib/python3.10/site-packages/urwid/event_loop/main_loop.py", line 337, in run
    self._run()
  File "/home/tom/.local/lib/python3.10/site-packages/urwid/event_loop/main_loop.py", line 439, in _run
    self.event_loop.run()
  File "/home/tom/.local/lib/python3.10/site-packages/urwid/event_loop/asyncio_loop.py", line 235, in run
    raise exc.with_traceback(exc.__traceback__)
  File "/usr/lib/python3.10/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/home/tom/.local/lib/python3.10/site-packages/urwid/event_loop/asyncio_loop.py", line 104, in wrapper
    return callback(*args, **kwargs)
  File "/home/tom/.local/lib/python3.10/site-packages/urwid/display/_posix_raw_display.py", line 273, in wrapper
    return self.parse_input(event_loop, callback, self.get_available_raw_input())
  File "/home/tom/.local/lib/python3.10/site-packages/urwid/display/_raw_display_base.py", line 488, in parse_input
    callback(decoded_codes, raw_codes)
  File "/home/tom/.local/lib/python3.10/site-packages/urwid/event_loop/main_loop.py", line 466, in _update
    self.process_input(keys)
  File "/home/tom/.local/lib/python3.10/site-packages/urwid/event_loop/main_loop.py", line 575, in process_input
    if hasattr(self._topmost_widget, "mouse_event") and self._topmost_widget.mouse_event(
  File "/home/tom/.local/lib/python3.10/site-packages/urwid/widget/frame.py", line 567, in mouse_event
    return self.body.mouse_event((maxcol, maxrow - htrim - ftrim), event, button, col, row - htrim, focus)
  File "/home/tom/.local/lib/python3.10/site-packages/urwid/widget/columns.py", line 1152, in mouse_event
    return w.mouse_event(w_size, event, button, col - x, row, focus)
  File "/home/tom/.local/lib/python3.10/site-packages/urwid/widget/listbox.py", line 1949, in mouse_event
    return not self._keypress_down((maxcol, maxrow))
  File "/home/tom/.local/lib/python3.10/site-packages/urwid/widget/listbox.py", line 1474, in _keypress_down
    self.change_focus((maxcol, maxrow), pos, row_offset, "above")
  File "/home/tom/.local/lib/python3.10/site-packages/urwid/widget/listbox.py", line 1182, in change_focus
    self._body.set_focus(position)
  File "/home/tom/.local/lib/python3.10/site-packages/urwid/widget/listbox.py", line 264, in set_focus
    self._modified()
  File "/home/tom/.local/lib/python3.10/site-packages/urwid/widget/listbox.py", line 98, in _modified
    signals.emit_signal(self, "modified")
  File "/home/tom/.local/lib/python3.10/site-packages/urwid/signals.py", line 298, in emit
    result |= self._call_callback(callback, user_arg, weak_args, user_args, args)
  File "/home/tom/.local/lib/python3.10/site-packages/urwid/signals.py", line 322, in _call_callback
    return bool(callback(*args))
  File "/home/tom/.local/lib/python3.10/site-packages/toot/tui/timeline.py", line 162, in modified
    self.draw_status_details(status)
  File "/home/tom/.local/lib/python3.10/site-packages/toot/tui/timeline.py", line 173, in draw_status_details
    self.status_details = StatusDetails(self, status)
  File "/home/tom/.local/lib/python3.10/site-packages/toot/tui/timeline.py", line 367, in __init__
    widget_list = list(self.content_generator(status.original, reblogged_by)
  File "/home/tom/.local/lib/python3.10/site-packages/toot/tui/timeline.py", line 485, in content_generator
    elif m["preview_url"].lower().endswith(('.jpg', '.jpeg', '.png', '.gif', '.svg', '.webp')):
AttributeError: 'NoneType' object has no attribute 'lower'

``` 

```
$ python3 -m toot env
Warning: Settings section [tui] has been deprecated in favour of [commands.tui].
toot 0.43.0
Python 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0]
Linux-4.4.0-19041-Microsoft-x86_64-with-glibc2.35
```
(this is via WSL)

**Comment by danschwarz** [comment]
Thanks, that's a fixable bug. I'll look into it.

**Comment by ihabunek** [comment]
Fixed in 0.44.0
